### PR TITLE
Cdutil average

### PR DESCRIPTION
**Pull request by jasonb5**
Changes:
- Replaces MV2.average with cdutil.average
- Adds weightoptions parameter

### CODE REVIEW

**Comment by doutriaux1** [comment]
build failed

**Comment by jasonb5** [comment]
test this jenkins

**Comment by jasonb5** [comment]
test this jenkins

**Comment by jasonb5** [comment]
test this jenkins

**Comment by jasonb5** [comment]
test this jenkins

**Comment by jasonb5** [comment]
test this jenkins

**Comment by jasonb5** [comment]
Can one of the admins verify this patch?

**Comment by jasonb5** [comment]
test this jenkins

**Comment by jasonb5** [comment]
test this jenkins

File: compute/wps/tasks/cdat.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import cdms2
 import cwt
+import cdutil
 from cdms2 import MV2 as MV
```
Comment (doutriaux1) [inline compute/wps/tasks/cdat.py:12]: just use
```python
import MV2
```

**Review by doutriaux1 — APPROVED** [review]

**Review by zshaheen — APPROVED** [review]
As you told me yesterday, I checked the results and all looks good. Nice Jenkins setup you have.

**Comment by jasonb5** [comment]
test this jenkins

**Pull request merged**